pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -156,6 +156,7 @@ func (p *cadvisorStatsProvider) ListPodStats(ctx context.Context) ([]statsapi.Po
 			podStats.CPU = cpu
 			podStats.Memory = memory
 			podStats.Swap = cadvisorInfoToSwapStats(podInfo)
+			podStats.IO = cadvisorInfoToIOStats(podInfo)
```

Comment:
can you also extend the cri_stats_provider and cri-api to have the structs needed for the changes to the summmary api?

Reply:
@haircommander Done. PTAL